src/Playwright.Tests/ScreencastTests.cs

```diff
@@ -21,9 +19,15 @@ public ScreencastTests(ITestOutputHelper output) : base(output)
         }
 
         [PlaywrightTest("screencast.spec.ts", "videoSize should require videosPath")]
-        [Fact(Skip = "We are not using old properties")]
-        public void VideoSizeShouldRequireVideosPath()
+        [SkipBrowserAndPlatformFact(skipWebkit: true, skipWindows: true)]
```

Comment:
Why are we skipping Windows? Is this an upstream thing as well? Just so we document it...

Reply:
done!